index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Hello Microverse!" />
+    <link href="style.css" rel="stylesheet">
+    <title>Hello Microverse!</title>
+</head>
+
+<body>
+
+    <h1>Hello Microverse!</h1>
+
```

Comment:
- [ ] [Optional] I would suggest you to use class name for the `<h1>` tag element, CSS classes enable you to apply unique style properties to groups of HTML elements to achieve your desired web page appearance also in another world  HTML class attribute specifies a class name. You can use it to select and access HTML elements for various tasks performed using CSS .. e.g ` <h1 class="header">Hello Microverse</h1>`